Makefile

```diff
@@ -15,6 +15,9 @@ test clean:
 
 test: all
 
+regex:
```

Comment:
We don't have this for other subdirectories. Why are we special-casing regex?

Reply:
Removed from the top level make file

Reply:
Removed from the top level make file